daemon/execdriver/native/driver.go

```diff
@@ -335,7 +335,9 @@ func (d *driver) Clean(id string) error {
 }
 
 func (d *driver) Stats(id string) (*execdriver.ResourceStats, error) {
+	d.Lock()
 	c := d.activeContainers[id]
+	d.Unlock()
```

Comment:
i think there're a few more occurrences of this in `Pause/Unpause`. We probably should apply the same fix. Wdyt ?